Cargo.toml

```diff
@@ -29,7 +29,21 @@ unsigned-varint = { version = "0.7.1", features = ["futures", "std"] }
 [dev-dependencies]
 async-std = { version = "1.10.0", features = ["attributes"] }
 env_logger = "0.9.0"
-libipld = { version = "0.15.0", default-features = false, features = ["dag-cbor"] }
-libp2p = { version = "0.50.0", features = ["tcp", "noise", "yamux", "rsa", "async-std"] }
-multihash = { version = "0.17.0", default-features = false, features = ["blake3", "sha2"] }
-tracing-subscriber = { version = "0.3.5", features = ["env-filter", "tracing-log"] }
+libipld = { version = "0.15.0", default-features = false, features = [
+    "dag-cbor",
+] }
+libp2p = { version = "0.51.0", features = [
+    "tcp",
+    "noise",
+    "yamux",
+    "rsa",
```

Comment:
Consider adding `ed25519` here as well, given that it won't be enabled by default in future versions.